redis/k8s-redis/redis-service.tf

```diff
@@ -0,0 +1,27 @@
+# This resource configures the redis service
+resource kubernetes_service redis_service {
```

Comment:
Maintain consistency in the code, the rest of the resource definitions are in quotes
`resource "kubernetes_service" "redis_service" { }`